src/main/resources/org/jraf/androidcontentprovidergenerator/abstractselection.ftl

```diff
@@ -34,9 +34,10 @@ public abstract class AbstractSelection<T extends AbstractSelection<?>> {
     private final StringBuilder mSelection = new StringBuilder();
     private final List<String> mSelectionArgs = new ArrayList<String>(5);
 
-    private Boolean mNotify;
-    private String mGroupBy;
-    private Integer mLimit;
+    Boolean mNotify;
```

Comment:
Because I access the value in other classes I generate for that package. You can replace with getters if you want.